### PR TITLE
Revising text and language throughout the app

### PR DESCRIPTION
**Pull request by alanna**

### CODE REVIEW

**Review by znarf — CHANGES_REQUESTED** [review]
All tests are failing so someone will need to dig into it.

**Comment by znarf** [comment]
First one is to run `npm run lint:quiet` or `npm run lint:fix` for the syntax issues. It's for example forbidden to a have single quote in a middle of string between single quotes. Usually, put the string between special quotes (instead of single quotes), this one: `

**Comment by alanna** [comment]
@znarf what do we need to do in order to merge this? I'm concerned it's going to get out of date with subsequent updates to the repo.

**Comment by alanna** [comment]
@znarf I think I fixed all the single quotes. Let me know if there's anything else needed.

File: components/discover/DiscoverCollectiveCard.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const DiscoverCollectiveCard = ({ collective, ...props }) => {
             {collective.stats.backers.all > 0 && (
               <FormattedMessage
                 id="discoverCard.backers.all"
-                defaultMessage="{count, plural, one {contributor {prettyCount} } other {Financial contributors {prettyCount} }}"
+                defaultMessage="{count, plural, one {contributor {prettyCount} } {Financial contributors {prettyCount} }}"
```
Comment (znarf) [inline components/discover/DiscoverCollectiveCard.js:25]: `other` is a keyword, not a a string that will appear for the user

**Comment by znarf** [comment]
I moved to a cleaner branch with a bunch of fixes but we're not there yet:
https://github.com/opencollective/opencollective-frontend/pull/5834

File: components/CreateOrganizationForm.js [hunk 2]
```diff
@@ -53,19 +53,19 @@ const orgMessages = defineMessages({
   },
   errorSlugHyphen: {
     id: 'createOrg.form.error.slug.hyphen',
-    defaultMessage: 'Organization slug can not start nor end with hyphen',
+    defaultMessage: 'Organization URL slug cannot start or end with a hyphen (-)',
   },
   errorWebsite: {
     id: 'createOrg.form.error.website',
-    defaultMessage: 'Enter valid website format www.test.com or test.org',
+    defaultMessage: 'Enter valid a website, e.g. www.example.com or example.org',
```
Comment (SudharakaP) [inline components/CreateOrganizationForm.js:60]: I think should be; 

```suggestion
    defaultMessage: 'Enter a valid website, e.g. www.example.com or example.org',
```

File: components/DefinedTerm.js [hunk 3]
```diff
@@ -53,26 +53,25 @@ const TranslatedDefinitions = defineMessages({
   [Terms.FISCAL_HOST]: {
     id: 'Fiscalhost.definition',
     defaultMessage:
-      'A fiscal host is a legal entity holding the money and responsible for the admin/taxes forms for the collective.',
+      'A Fiscal Host is an entity that holds the money on behalf of a Collective, and taxes care of accounting, taxes, and invoices.',
```
Comment (SudharakaP) [inline components/DefinedTerm.js:56]:
```suggestion
      'A Fiscal Host is an entity that holds the money on behalf of a Collective, and takes care of accounting, taxes, and invoices.',
```

**Comment by znarf** [comment]
@SudharakaP for your feedback, can you share your comments on the latest PR?

https://github.com/opencollective/opencollective-frontend/pull/5834

**Pull request closed without merging**